torch/_torch_docs.py

```diff
@@ -3894,6 +3894,10 @@ def merge_dicts(*dicts):
 (see :func:`torch.squeeze`), resulting in both the :attr:`values` and
 :attr:`indices` tensors having 1 fewer dimension than the :attr:`input` tensor.
 
+.. note::
+    When there are multiple valid :attr:`k` th values, this function may
```

Comment:
Only if the input is CUDA, though. 

"When :attr:\`input\` is on CUDA and there are multiple valid :attr:\`k\`th values, this function may nondeterministically return :attr:\`indices\` for any of them."